latch/ldata/_transfer/upload.py

```diff
@@ -124,6 +124,12 @@ def upload(
                     for file_name in file_names:
                         rel_path = Path(dir_path) / file_name
 
+                        try:
+                            total_bytes += rel_path.stat().st_size
+                        except FileNotFoundError:
+                            print(f"WARNING: file {rel_path} not found, skipping...")
+                            continue
+
```

Comment:
```suggestion
                        p_stat = rel_path.stat()
                        if not stat.S_ISREG(p_stat.st_mode):
                            print(f"WARNING: file {rel_path} not found, skipping...")
                            continue

```

Reply:
This will fail with `FileNotFound` (same issue as before) because stat will fail